shared/domains/chelonia/chelonia.js

```diff
@@ -344,25 +471,28 @@ export default (sbp('sbp/selectors/register', {
     }
   },
   'chelonia/latestContractState': async function (contractID: string) {
-    const events = await sbp('chelonia/out/eventsSince', contractID, contractID)
-    let state = {}
+    const events = await sbp('chelonia/private/out/eventsSince', contractID, contractID)
+    if (sbp(this.config.stateSelector)[contractID]) {
+      return cloneDeep(sbp(this.config.stateSelector)[contractID])
+    }
```

Comment:
The call to `'chelonia/private/out/eventsSince'` should happen after this `if` condition, right?

Reply:
Good catch. Perhaps, will test it.

Reply:
Yes,  it should.